Index: src/test/java/org/example/pages/First_Step/P07_ConfirmationSignatures.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package org.example.pages.First_Step;\r\n\r\nimport org.example.stepDefs.Hooks;\r\nimport org.openqa.selenium.By;\r\nimport org.openqa.selenium.WebElement;\r\n\r\npublic class P07_ConfirmationSignatures {\r\n\r\n   public WebElement AGREEMENT() {\r\n       return Hooks.driver.findElement(By.xpath(\"//img[@ng-src='img/AGREEMENT.png']\"));\r\n   }\r\n\r\n    public WebElement AGREEMENT_label()\r\n    {\r\n        return Hooks.driver.findElement(By.id(\"referrer\"));\r\n    }\r\n\r\n    public WebElement SUPPLY() {\r\n        return Hooks.driver.findElement(By.xpath(\"//img[@ng-src='img/SUPPLY.png']\"));\r\n    }\r\n\r\n    public WebElement SUPPLY_label()\r\n    {\r\n        return Hooks.driver.findElement(By.id(\"referrer\"));\r\n    }\r\n\r\n\r\n\r\n\r\n}\r\n
===================================================================
diff --git a/src/test/java/org/example/pages/First_Step/P07_ConfirmationSignatures.java b/src/test/java/org/example/pages/First_Step/P07_ConfirmationSignatures.java
--- a/src/test/java/org/example/pages/First_Step/P07_ConfirmationSignatures.java	
+++ b/src/test/java/org/example/pages/First_Step/P07_ConfirmationSignatures.java	
@@ -10,7 +10,7 @@
        return Hooks.driver.findElement(By.xpath("//img[@ng-src='img/AGREEMENT.png']"));
    }
 
-    public WebElement AGREEMENT_label()
+    public WebElement Treatment_label()
     {
         return Hooks.driver.findElement(By.id("referrer"));
     }
@@ -19,12 +19,6 @@
         return Hooks.driver.findElement(By.xpath("//img[@ng-src='img/SUPPLY.png']"));
     }
 
-    public WebElement SUPPLY_label()
-    {
-        return Hooks.driver.findElement(By.id("referrer"));
-    }
-
-
 
 
 }
